config-importer/src/main/scala/Importer.scala

```diff
@@ -0,0 +1,358 @@
+
+  // val autoAddBlankBands = List(
+  //   ""
+  // )
+
+  val autoExciseBands = List(
+    "AboveAUST", "AboveUST"
+  )
+
+  def rateBlockOld(
+    bandName : String,
+    data: Map[String,Map[BigDecimal,String]],
+    exceptions: List[String]
+  ) : String = {
+    val exceptionsBlock = exceptions match {
+      case Nil => None
+      case x => Some("gross-pay-exceptions: [\n" ++ x.mkString(",\n").indent(1) ++ "\n]")
+    }
+
+    bandNameConversion(bandName) is {
+      List(
+        data.get("EE").map{ee => subBlock(None, "employee", ee)},
+        data.get("ER").map{er => subBlock(None, "employer", er)},
+        exceptionsBlock
+      ).flatten.mkString("\n")
+    }
+  }
+
+  def rateBlock(
+    bandName : String,
+    data: Map[String,Map[BigDecimal,String]]
+  ) : String = bandName match {
+    case "Rate" => rateBlock("PT to UEL", data.filter(_._1 == "EE")) ++ "\n" ++
+      rateBlock("ST to UEL", data.filter(_._1 == "ER"))
+    case _ =>
+      (data.get("EE"), data.get("ER")) match {
+        case (Some(ee),Some(er)) =>
+          bandNameConversion(bandName) is {
+            subBlock(None, "employee", ee) + "\n" +
+            subBlock(None, "employer", er) 
+          }
+        case (Some(ee),None) => subBlock(Some(bandName), "employee", ee)
+        case (None, Some(er)) => subBlock(Some(bandName), "employer", er)
+        case _ => bandNameConversion(bandName) + " { }"
+      }
+  }
+
+  def limitsOld(bands: Map[String, String]): String = "limits" is {
+    // up to 5 'bands' - Wk, Mnth and Ann values, e.g. - 'Ann Band 4'
+    // 'bands' 1 and 5 always present
+    // 'bands' 2-4 present when year > 1984
+    // these are actually limits, not bands, despite what the access db says
+    val indexedLimits = (1 to 5).map { i =>
+      (
+        bands.get(s"Wk Band $i").map(BigDecimal.apply),
+        bands.get(s"Mnth Band $i").map(BigDecimal.apply),
+        bands.get(s"Ann Band $i").map(BigDecimal.apply),
+      ).tupled
+    }.toList.flatten
+
+    val limitNames = indexedLimits.size match {
+      case 2 => Vector("LEL", "UEL")
+      case 5 => Vector("LEL", "", "", "", "UEL")
+      case _ => sys.error(s"I don't know how to label the bands")
+    }
+
+    val limits = (limitNames zip indexedLimits).filter(_._1.nonEmpty)
+
+    limits.map { case (k, (weekBD, mnthBD, year)) =>
+      val month = Some(mnthBD).filter(_ != (year / 12).setScale(0, HALF_UP))
+      val week = Some(weekBD).filter(_ != (year / 52).setScale(0, HALF_UP))
+
+      if(month.isEmpty && week.isEmpty) {
+        s"$k.year: ${year.formatMoney}"
+      } else {
+        k is {List(
+          Some(s"year: ${year.formatMoney}"),
+          month.map(x => s"month: ${x.formatMoney}"),
+          week.map(x => s"week: ${x.formatMoney}")
+        ).flatten.mkString("\n")}
+      }
+    }.mkString("\n")
+  }
+
+  def correctLimitNames(in: String): String = in match {
+    case "EE_ET" => "PT"
```

Comment:
it appears that `PT` and `ST` were introduced from the 2011 tax year - before that it was just `ET`

Reply:
Well there was EE_ET and ER_ET which can in some cases be merged to ET. 

Time allowing I do intend to go back and make these changes, however as the EE_ET is effectively the PT and the ER_ET is the ST and this is just labeling having no effect on the calculations I'd like to do that afterwards. 

There are some other things that can be done regarding the config file (for example stripping out the limits for UD where they're not needed), but I'm conscious that this branch is already getting quite extensive!